### DISCUSSION TITLE
Does Erasure Coding support separate storage

### DISCUSSION THREAD

**Discussion by fengjiannan2010** [minio/minio#16254] [Q&A] [answered]
Does Erasure Coding support separate storage of data and verification. For example, I configure 6 disks here, 5 disks store the original data, and 1 disk stores the verification data?

**Comment by klauspost** [comment, marked as answer]
Distribution is done on a per-object basis to spread load and support multiple storage classes.

Therefore there are no dedicated drives for data or erasure codes.
